### PR TITLE
create OptimumDNA and Individual classes

### PR DESCRIPTION
**Pull request by yfenes**
First pull request.

_Edit after the comment below:_ 

I have set the Django and SQLite database.
I have created models for genetic app. These are:
- OptimumDNA
- Individual

where there is a one to many relation. **Many Individual** may be evaluated/scored wrt **one OptimumDNA**.

### CODE REVIEW

**Comment by SaitTalhaNisanci** [comment]
You should probably write more descriptive message instead of `First Pull Request`.

File: .idea/geneticApp.iml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (SaitTalhaNisanci) [inline .idea/geneticApp.iml]: This file like the following files should not be in the repository. You should put `.idea` to `.gitignore` to ignore such files.

Comment (SaitTalhaNisanci) [reply .idea/geneticApp.iml]: Also dont include the binaries here like the above one.

File: geneticappenv/bin/activate [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# This file must be used with "source bin/activate" *from bash*
```
Comment (SaitTalhaNisanci) [inline geneticappenv/bin/activate]: You should not push your virtual env. add this to `.gitignore`

File: training/genetic/models.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    DNA = models.CharField(max_length=100)
+    #pub_date = models.DateTimeField('date published')
+
+    def __str__(self):
+        return self.DNA
+
+class Individual(models.Model):
+
+    optimum_DNA = models.ForeignKey(OptimumDNA, on_delete=models.CASCADE)
+    DNA = models.CharField(max_length=100)
```
Comment (SaitTalhaNisanci) [inline training/genetic/models.py:17]: Why do you have a DNA here and in OptimumDNA, if they will be the same DNAS this one is redundant.

File: training/genetic/models.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        return self.DNA
+
+    # TODO the argument opt always must be the OptimumDNA associated with this Individual
+    def fitness(self, optDNA):
+        sen1 = self.DNA
+        sen2 = optDNA
+
+        if len(sen1) != len(sen2):
+            print("Error, length of sentences must be the same.\nSentence 1: " + sen1 + "\nSentence 2: " + sen2)
+            return 0, 0
```
Comment (SaitTalhaNisanci) [inline training/genetic/models.py]: What if the score is actually 0 and then percentage will be 0. If you return `0, 0` we cannot know that. Maybe throw an error or return `-1,-1`

File: training/genetic/models.py (hunk 3 above)
Comment (yfenes) [reply training/genetic/models.py:17]: In the population, there are Individuals. OptimumDNA is the ultimate DNA for the population, it is not an Individual. 

*OptimumDNA* is a DNA that Individuals may relate to.
*DNA* is DNA of the Individual, there can be many Individuals.

For example:
OptimumDNA of a population : 
 - "Hello world"

Individuals of the population : 
 - "Hello corld"
 - "Yello world"
 - "Yello corld"

File: training/genetic/models.py (hunk 4 above)
Comment (yfenes) [reply training/genetic/models.py]: Will be done 👍

**Review by SaitTalhaNisanci — APPROVED** [review]
LGTM

**Pull request merged**